### DISCUSSION TITLE
Focus behavior for windows with 100%+ width or height.

### DISCUSSION THREAD

**Discussion by salman-farooq-sh** [YaLTeR/niri#516] [Ideas]
If a window is 100%+ in width or height, you can't put the right or bottom side that sticks out into the viewport.

I think we need a way to {top-bottom-left-right}-edge-snap the 100%+ width/height windows to the corresponding monitor edge, possibly with the same Mod+HJKL, or maybe even scroll freely in both dimensions.

Edge-snapping will omit the center area of windows that are more than 200% width/height.

Maybe snap in intelligent stages for ~150%+?

**Comment by YaLTeR** [comment]
I'm not sure how this issue is any different from the second one you made?

**Reply by salman-farooq-sh** [reply]
https://github.com/YaLTeR/niri/issues/514 is for multiple windows in columns only. This is very similar but specialized for a single window (intelligent staged snapping etc.) and also talks about width instead of just height.
